code/modules/reagents/reagent_containers/food/snacks/fish.dm

```diff
@@ -101,3 +101,14 @@
 		crack_shell(user)
 		return
 	return ..()
+
+/obj/item/mollusc/clam/rasval
```

Comment:
Can you make a seed for this as well and add it to the botany vending machine?

Reply:
Done, let me know if it should only be available in the one for the hydroponics job rather than the public one as well.